nmostesting/suites/IS0401Test.py

```diff
@@ -1410,7 +1410,7 @@ def test_20_01(self, test):
                     headers = None
                 valid, response = self.do_request("GET", href, headers=headers)
                 if valid and response.status_code == 200:
-                    valid, message = check_content_type(response.headers, "application/sdp")
+                    valid, message = check_content_type(response.headers, "application/sdp")
```

Comment:
Not clear in GitHub view what the difference is on this line or others in this file. Please revert all changes to this file.